packages/cli/src/commands/build.ts

```diff
@@ -608,6 +651,59 @@ async function doBuild(
   );
 }
 
+export async function determineFrameworkVersion(
```

Comment:
Should this be `detectFrameworkVersion` to match the other existing function names?

Reply:
Updated in https://github.com/vercel/vercel/pull/9009#:~:text=Resolve%20conversation-,PR%20feedback,-fbf562e

Reply:
That number is the same as this PR 😄 

Is that a copy/pasta error?

Reply:
Huh, yeah. This is the right place: [`fbf562e` (#9009)](https://github.com/vercel/vercel/pull/9009/commits/fbf562eb51e1fb296ecbde2e4fdab807bb069fad)